contracts/staking/PoSValidatorManager.sol

```diff
@@ -356,7 +364,11 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
 
         Delegator memory delegator = $._delegatorStakes[delegationID];
         _unlock(delegator.weight, delegator.owner);
-        // TODO: issue rewards
+
+        uint256 rewardAmount = $._rewardCalculator.calculateReward(
```

Comment:
Same here, regarding the TODO on line 280

Reply:
as discussed in https://github.com/ava-labs/teleporter/pull/534#discussion_r1754707932 ,
addressed in https://github.com/ava-labs/teleporter/pull/534/commits/57a519e728da4431a644ebd148f588bad396ad9e